activities/codebase-stewardship/activities.md

```diff
@@ -6,10 +6,12 @@ type: Resource
 
 We provide codebase stewardship at the codebase level. Stewardship activities take place in 4 domains:
 
-* community
-* product marketing
-* quality
-* support
+* [community](#community)
+* [product](#product)
```

Comment:
Is this pre-empting (or potentially solving) #642 ?

Reply:
It was not my intention, I merely harmonized the list here with the headers below.